packages/core/stories/patterns/contact-details/contact-details.stories.tsx

```diff
@@ -0,0 +1,270 @@
+      direction={"row"}
+      gap={2}
+      style={{
+        width: "400px",
+      }}
+    >
+      <Avatar
+        src={avatarSrc as string}
+        aria-label={basicContact.primary}
+        fallbackIcon={<UserBadgeIcon />}
+        size={2}
+      />
+      <StackLayout direction={"column"} gap={1}>
+        <Text styleAs="h3">{basicContact.primary}</Text>
+        <Text styleAs="h4">{basicContact.secondary}</Text>
+        <StackLayout direction={"row"} gap={2}>
+          <StackLayout gap={1} direction={"column"}>
+            {basicContact.metadata.map((metadata) => {
+              return (
+                <Text variant="secondary" key={metadata.label}>
+                  {metadata.label}
+                </Text>
+              );
+            })}
+          </StackLayout>
+          <StackLayout gap={1} direction={"column"}>
+            {basicContact.metadata.map((metadata) => {
+              return <Text key={metadata.label}>{metadata.value}</Text>;
+            })}
+          </StackLayout>
+        </StackLayout>
+      </StackLayout>
+    </StackLayout>
+  );
+};
+
+export const Horizontal = () => {
+  return (
+    <StackLayout direction={"row"} gap={2} style={{ width: "400px" }}>
+      <Avatar
+        src={persona as string}
+        aria-label={basicContact.primary}
+        fallbackIcon={<UserBadgeIcon />}
+        size={2}
+      />
+      <StackLayout direction={"column"} gap={1}>
+        <Text styleAs="h3">{basicContact.primary}</Text>
+        <Text styleAs="h4">{basicContact.secondary}</Text>
+        <FlowLayout gap={1}>
+          {basicContact.metadata.map((metadata) => {
+            return (
+              <FlexLayout
+                key={metadata.label}
+                align="center"
+                justify="center"
+                gap={1}
+              >
+                {metadata.icon}
+                <Text key={metadata.label} aria-label={metadata.label}>
+                  {metadata.value}
+                </Text>
+              </FlexLayout>
+            );
+          })}
+        </FlowLayout>
+      </StackLayout>
+    </StackLayout>
+  );
+};
+
+export const WithIcon = () => {
+  return <ContactDetails avatarSrc="" />;
+};
+
+export const CardEmbedded = () => {
+  return (
+    <Card>
+      <ContactDetails />
+    </Card>
+  );
+};
+
+export const QuickAction = () => {
+  return (
+    <StackLayout direction="row" gap={2} style={{ width: "400px" }}>
+      <Avatar
+        src={persona as string}
+        aria-label={basicContact.primary}
+        fallbackIcon={<UserBadgeIcon />}
+        size={2}
+      />
+      <StackLayout direction={"column"} gap={1}>
+        <Text styleAs="h3">{basicContact.primary}</Text>
+        <Text styleAs="h4">{basicContact.secondary}</Text>
+        <StackLayout gap={2} direction={"row"}>
+          <Tooltip
+            content={`Email ${basicContact.primary}`}
+            placement={"bottom"}
+          >
+            <Button variant="secondary">
```

Comment:
i think we will also need an aria-label to describe purpose here as this Button contains only an icon and no text